### PR TITLE
Some typo fixes

### PR DESCRIPTION
**Pull request by afredooo**
Just some minor typo fixes for better FAQ reading.
Thanks!

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5339) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=5339) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=5339) it.</sub>

File: docs/how-to-use-and-FAQ/FAQ-ncnn-produce-wrong-result.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### caffemodel should be row-major
 
-`caffe2ncnn` tool assumes the caffemodel is row-major (produced by c++ caffe train command).
+`caffe2ncnn` tool asumes the caffemodel is row-major (produced by c++ caffe train command).
```
Comment (nihui) [inline docs/how-to-use-and-FAQ/FAQ-ncnn-produce-wrong-result.md:3]: why is `assumes` a typo ?

Comment (afredooo) [reply docs/how-to-use-and-FAQ/FAQ-ncnn-produce-wrong-result.md]: This was a mistake, my bad! I corrected it to be ready for the merge.
Have a nice day!

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**